docs/_static/C++/CMakeLists.txt

```diff
@@ -0,0 +1,129 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Aha. You are trying to include this example program into docs/_static folder. How about moving this into Open3D/examples in the future?

Reply:
I think that's potentially a great idea, I didn't know where to put it. We need to discuss this more offline.

And you just gave me an idea: why not configure cmake install to install the documentation files as well? We can do this as a separate PR. What do you think?

Reply:
Sounds good idea. Is the proposal intended for CI?

Reply:
True, they wouldn't need to. In light of that my idea is incomplete.

I was thinking more about how we would go about packaging Open3D in the future - we might want to put some documentation inside the package. However I agree, the form that documentation might take might be different from this one.